app/models/homepage.rb

```diff
@@ -2,4 +2,8 @@ class Homepage < ApplicationRecord
   belongs_to :user
 
   has_and_belongs_to_many :lists
+
+  def lists *args
+    super(*args).uniq
```

Comment:
Yeah, a lot of the activerecord query methods were returning one list per listmembership, and there can be multiple memberships on one list...so I had to make sure they're unique.